runtime/parachains/src/paras_inherent.rs

```diff
@@ -1061,24 +1053,91 @@ mod tests {
 				assert_eq!(<scheduler::Pallet<Test>>::scheduled(), vec![]);
 
 				// Nothing is filtered out (including the backed candidates.)
-				let limit_para_inherent_data = Pallet::<Test>::create_inherent_inner(&inherent_data.clone()).unwrap();
-				assert!(limit_para_inherent_data != expected_para_inherent_data);
+				let multi_dispute_inherent_data = Pallet::<Test>::create_inherent_inner(&inherent_data.clone()).unwrap();
+				assert!(multi_dispute_inherent_data != expected_para_inherent_data);
+
+				assert_eq!(multi_dispute_inherent_data.disputes.len(), 2);
 
 				assert_eq!(
-					limit_para_inherent_data.disputes.len(),
-					2usize,
+					&multi_dispute_inherent_data.disputes[..2],
```

Comment:
Can this just be `&multi_dispute_inherent_data.disputes` since it looks like it should only have 2 disputes?

Reply:
Yes, but I think it's more explicit to slice it on the same bound